pkg/scaffold/internal/templates/v2/main.go

```diff
@@ -48,6 +53,23 @@ func (f *Main) SetTemplateDefaults() error {
 	return nil
 }
 
+var hasher = fnv.New32a()
+
+func hash(s string) (string, error) {
+	if _, err := hasher.Write([]byte(s)); err != nil {
```

Comment:
This `Write` method will never return an error.

Reply:
Yay, but this is to make the linter happy.
I probably can use `// nolint` to disable the checking for this line.